unlock-app/package.json

```diff
@@ -3,19 +3,25 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/core": "^7.1.2",
     "bn.js": "^4.11.8",
     "bootstrap": "^4.0.0",
+    "color": "^0.11.0",
     "color-scheme": "^1.0.1",
+    "next": "^7.0.2",
+    "path-match": "^1.2.4",
     "prop-types": "^15.6.1",
-    "react": "^16.4.2",
-    "react-dom": "^16.4.2",
+    "react": "^16.5.2",
+    "react-app-polyfill": "^0.1.3",
+    "react-dom": "^16.5.2",
     "react-jazzicon": "^0.1.2",
+    "react-no-ssr": "^1.1.0",
     "react-redux": "^5.0.7",
     "react-router-dom": "^4.2.2",
     "react-router-redux": "^5.0.0-alpha.9",
-    "react-scripts": "^2.0.4",
     "react-test-renderer": "^16.3.2",
-    "redux": "^4.0.1",
+    "redux": "^3.7.2",
+    "redux-devtools-extension": "^2.13.5",
```

Comment:
Wouldn't that be a `devDependency` ? That said a lot of dependencies in here are probably dev... probably worth a cleanup at some point.